cloudbuildMaster.yaml

```diff
@@ -1,7 +1,11 @@
 steps:
- - name: maven:3-jdk-8
-   entrypoint: make
+ - name: 'gcr.io/cloud-builders/docker'
+   args: ['build', '-t', 'gcr.io/step-team-145/make', '.']
+ - name: 'gcr.io/step-team-145/make'
    args: ['devprep']
  - name: maven:3-jdk-8
    entrypoint: mvn
-   args: ['package', 'appengine:deploy']
\ No newline at end of file
+   args: ['package', 'appengine:deploy']
```

Comment:
out of curiosity - can this be:

```
entrypoint: gcr.io/step-team-145/make
args: ['deploy']
```

That way you only have to maintain the proper arguments in one place.

Reply:
Before, the docker image didn't support maven, but I just changed the image to another one and this seems to work !!